src/metabase/automagic_dashboards/comparison.clj

```diff
@@ -2,111 +2,119 @@
   (:require [clojure.string :as str]
```

Comment:
Is this file still needed? It looks like everything is now commented out

Reply:
It's temporary, because I kept breaking it. But maybe it would be better to pull it out until we get to it.